clients/src/test/java/org/apache/kafka/clients/consumer/CooperativeStickyAssignorTest.java

```diff
@@ -44,14 +46,26 @@ public AbstractStickyAssignor createAssignor() {
     }
 
     @Override
-    public Subscription buildSubscription(List<String> topics, List<TopicPartition> partitions) {
-        return new Subscription(topics, assignor.subscriptionUserData(new HashSet<>(topics)), partitions);
+    public Subscription buildSubscriptionV0(List<String> topics, List<TopicPartition> partitions, int generationId) {
+        // cooperative sticky assignor only supports ConsumerProtocolSubscription V1 or above
+        return null;
```

Comment:
nit: If it should not be used, would it make sense to throw an exception? e.g. IllegalStateException

Reply:
Thanks David, but in `testMemberDataFromSubscription` test, I did use `null` check to make sure we will skip ConsumerProtocolSubscription v0 in cooperativeStickyAssignor. We can throw exception here, but that would make the test more complicated and that's unnecessary IMO.